lambda/css-api/src/routes.ts

```diff
@@ -285,7 +285,7 @@ export const setRoutes = (app: any) => {
       }
       #swagger.parameters['roleName'] = {
```

Comment:
For these ones with :roleName, I was just thinking to add a block:

      #swagger.responses[400] = {
        description: 'Not Found',
        schema: { message: 'string' }
      }
      
      I just noticed it when testing that the response is undocumented, not directly from this change